.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest]
+        os: [ubuntu-latest, macOS-latest, windows-latest]
         cabal: ["3.6"]
         ghc:
           - "8.10.7"
```

Comment:
Let's also add the `exclude` section similar to this one:

* https://github.com/co-log/co-log-core/blob/885723b539ea352164982914aab09d499033c400/.github/workflows/ci.yml#L29-L54

The idea is that there's no much benefit in building on Ubuntu, macOS and Windows with all 3 GHC versions (resulting in 9 total builds). We can only buid with 3 versions on Ubuntu and use the latest GHC version on Windows and macOS (resulting in 5 builds in total).

This increases CI speed significantly without losing anything important.